server/routes/routerRequestHandlers.ts

```diff
@@ -1,19 +1,26 @@
 import { NextFunction, Request, Response, RequestHandler } from 'express'
 import createError from 'http-errors'
-import { CiagService } from '../services'
+import { CiagService, InductionService } from '../services'
 import logger from '../../logger'
+import config from '../config'
+import toCiagPlan from '../data/mappers/ciagPlanMapper'
 
 /**
- *  Middleware function that returns a Request handler function to retrieve the Induction via the [CiagService] and
- *  store in the express request context.
+ *  Middleware function that returns a Request handler function to retrieve the [InductionDto] via the [InductionService]
+ *  and store in the express request context as a [CiagPlan].
  */
-const retrieveInduction = (ciagService: CiagService): RequestHandler => {
+const retrieveInduction = (ciagService: CiagService, inductionService: InductionService): RequestHandler => {
   return async (req: Request, res: Response, next: NextFunction) => {
     const { id: prisonNumber } = req.params
     const { user } = res.locals
 
     try {
-      req.context.plan = await ciagService.getCiagPlan(user.token, prisonNumber)
+      if (config.featureToggles.useNewInductionApiEnabled) {
```

Comment:
Because the cypress tests have the feature toggle enabled (via the change in `feature.env`) all the cypress tests that need to get and load and induction will be doing so via the Induction Service 👍